code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -11,36 +11,93 @@
 	name = "flesh"
 	flags = NODROP
 
+/obj/item/clothing/glasses/changeling/attack_hand(mob/user)
+	if(loc == user && user.mind && user.mind.changeling)

```

Comment:
That would allow other people / things remove them, not that being unable to remove someone's shoes is good changeling clue.